packages/hub/src/utils/sha1.ts

```diff
@@ -0,0 +1,31 @@
+import { hexFromBytes } from "./hexFromBytes";
+import { isFrontend } from "./isFrontend";
+
+export async function* sha1(buffer: Blob, opts?: { abortSignal?: AbortSignal }): AsyncGenerator<number, string | null> {
+	yield 0;
+
+	if (globalThis.crypto?.subtle) {
```

Comment:
no threshold like for sha256 since non-LFS files are always < 10MB (which is the threshold for sha256 IIUC)